osu.Game/Overlays/Settings/Sections/GraphicsSection.cs

```diff
@@ -25,6 +25,7 @@ public GraphicsSection()
                 new LayoutSettings(),
                 new RendererSettings(),
                 new ScreenshotSettings(),
+                new VideoSettings(),
```

Comment:
Would feel more organized to put the video settings next to the renderer, rather than having the screenshot section inbetween.

```suggestion
                new VideoSettings(),
                new ScreenshotSettings(),
```

Reply:
I would tend to agree but not a huge deal I don't think.

Reply:
Let's go with the suggestion. I believe it makes more sense.